Improved error messaging for `remove`

### Environment

uds v0.9.0

### Steps to reproduce

1. Run `uds remove` on an invalid bundle

### Expected result

Error messaging indicating that you're attempting to run `remove` with an invalid bundle name/path

### Actual Result
```
panic: runtime error: slice bounds out of range [:6] with length 4

goroutine 1 [running]:
github.com/defenseunicorns/uds-cli/src/pkg/utils.EnsureOCIPrefix(...)
        /home/runner/work/uds-cli/uds-cli/src/pkg/utils/oci.go:250
github.com/defenseunicorns/uds-cli/src/pkg/bundle.getOCIValidatedSource({0x16bbb2ea8, 0x4})
        /home/runner/work/uds-cli/uds-cli/src/pkg/bundle/remote.go:263 +0x420
github.com/defenseunicorns/uds-cli/src/pkg/bundle.CheckOCISourcePath({0x16bbb2ea8, 0x4})
        /home/runner/work/uds-cli/uds-cli/src/pkg/bundle/remote.go:330 +0x40
github.com/defenseunicorns/uds-cli/src/pkg/bundle.(*Bundle).Remove(0x140017a6000)
        /home/runner/work/uds-cli/uds-cli/src/pkg/bundle/remove.go:28 +0x38
github.com/defenseunicorns/uds-cli/src/cmd.glob..func13(0x10b9b9d40?, {0x140003f3aa0?, 0x4?, 0x10754ba68?})
        /home/runner/work/uds-cli/uds-cli/src/cmd/uds.go:129 +0x158
github.com/spf13/cobra.(*Command).execute(0x10b9b9d40, {0x140003f3a40, 0x2, 0x2})
        /home/runner/go/pkg/mod/github.com/spf13/cobra@v1.8.0/command.go:987 +0x828
github.com/spf13/cobra.(*Command).ExecuteC(0x10b9ba8c0)
        /home/runner/go/pkg/mod/github.com/spf13/cobra@v1.8.0/command.go:1115 +0x344
github.com/spf13/cobra.(*Command).Execute(...)
        /home/runner/go/pkg/mod/github.com/spf13/cobra@v1.8.0/command.go:1039
github.com/defenseunicorns/uds-cli/src/cmd.Execute()
        /home/runner/work/uds-cli/uds-cli/src/cmd/root.go:60 +0x24
main.main()
        /home/runner/work/uds-cli/uds-cli/main.go:10 +0x1c
```

--- a/src/pkg/utils/oci.go
+++ b/src/pkg/utils/oci.go
@@ -11,6 +11,7 @@ import (
 	"errors"
 	"fmt"
 	"slices"
+	"strings"
 
 	"github.com/defenseunicorns/uds-cli/src/config"
 	"github.com/defenseunicorns/uds-cli/src/types"
@@ -247,7 +248,7 @@ func GetIndex(remote *oci.OrasRemote, ref string) (*ocispec.Index, error) {
 // EnsureOCIPrefix ensures oci prefix is part of provided remote source path, and adds it if it's not
 func EnsureOCIPrefix(source string) string {
 	var ociPrefix = "oci://"
-	if source[:len(ociPrefix)] != ociPrefix {
+	if !strings.Contains(source, ociPrefix) {
 		return ociPrefix + source
 	}
 	return source

